release/_releases_json.sh

```diff
@@ -119,5 +119,5 @@ function _promote_product_versions {
 		else
 			lc_log INFO "No product version found to promote for ${product_name}-${group_version}."
 		fi
-	done < "${_RELEASE_ROOT_DIR}/supported-${product_name}-versions.txt"
+	done < <(grep "" "${_RELEASE_ROOT_DIR}/supported-${product_name}-versions.txt")
```

Comment:
@natocesarrego , This way the `while read` can go trough the file without needing an enter at the end of the file. Otherwise `while read` can only read the lines if it is terminated by an enter.

Reply:
@Wykenz, please reach me out on next Monday to discuss about this pull

Thanks

Reply:
@Wykenz, can you please adjust this solution based on https://stackoverflow.com/questions/12916352/shell-script-read-missing-last-line?

Reply:
@natocesarrego I pushed up the changes